### ISSUE TITLE
no_throwing_strings rule does not trigger when using round brackets

### ISSUE THREAD

**Issue by Ionaru** [clutchski/coffeelint#639]
The no_throwing_strings rule does not trigger when using round brackets.

Consider the following code:
```coffeescript
test ->
  throw 'moo'  # Lint fails (correct)
```
```coffeescript
test ->
  throw('moo')  # Lint passes (incorrect)
```
```coffeescript
test ->
  throw ('moo')  # Lint passes (incorrect)
```

This is strange because the resulting javascript is the same:
```js
test(function() {
  throw 'moo';
});
```
